TerrariaServerAPI/TerrariaApi.Server/ServerApi.cs

```diff
@@ -22,6 +23,7 @@ public static class ServerApi
 		private static Main game;
 		private static readonly Dictionary<string, Assembly> loadedAssemblies = new Dictionary<string, Assembly>();
 		private static readonly List<PluginContainer> plugins = new List<PluginContainer>();
+		private static readonly Dictionary<string, Version> dangerousPlugins = new Dictionary<string, Version>();
```

Comment:
Can we add documentation on this member, specifically, what the key and value represent?
It is obvious once you get to the MaliciousPlugin check but until then, this field makes no sense.
I would indicate what the key represents (A hash of something) and what the value represents (the latest version known to be dangerous).

Reply:
@Olink I already added a commit that addresses this xoxo